testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -1021,7 +1021,16 @@ QUnit.module('Lookup', {
         mouse.start().down().move(0, 10).up();
 
         openPopupWithList(firstLookup);
-        assert.equal($('.' + LIST_ITEM_SELECTED_CLASS).length, 2);
+
+        // NOTE: in ShadowDOM mode one selected item is inside ShadowDOM
+        // and other is in document
+        if(QUnit.isInShadowDomMode()) {
+            assert.equal(document.querySelectorAll('.' + LIST_ITEM_SELECTED_CLASS).length, 1);
```

Comment:
```suggestion
            assert.strictEqual(document.querySelectorAll(`.${LIST_ITEM_SELECTED_CLASS}`).length, 1);
```

and similar for near rows